setup.py

```diff
@@ -7,16 +7,45 @@
 here = pathlib.Path(__file__).parent.resolve()
 long_description = open("README.md").read()
 
-# Requirements
+##################
+## Requirements ##
+##################
 
-# Explicitly add optional dependencies for conda compatiblity,
-# for instance, avoid using fastapi[all]
 install_requires = open("requirements.txt").read().strip().split("\n")
 dev_requires = open("dev-requirements.txt").read().strip().split("\n")
 
-# Human-Readable/Reusable Extras
-# Add these to `optional-requirements.txt` as well
+# Human-Readable Extras
+# Add these to `optional-requirements.txt` as well for Docker caching
+aws = ["boto3~=1.24.46"]
```

Comment:
See previous comment. I think this will become required with email functionality.

Reply:
Not particularly worried about it now then, we'll just have to handle it when we do another merge of fidesops